terra-jupyter-r/Dockerfile

```diff
@@ -37,6 +37,11 @@ RUN mkdir -p /home/jupyter-user/.rpackages \
     "remotes", \
     "BiocManager", \
     "devtools"))' \
+
+    # fix for AoU-blocking bug https://github.com/rstudio/reticulate/issues/693
+    # after version 1.15 is released to CRAN, we can install reticulate in the standard way
+    && R -e 'devtools::install_github("rstudio/reticulate", ref="00172079")' \
```

Comment:
I don't know a lot about R packages. @nturaga does this look right to you or should we be using `BiocManager`?

Reply:
This is a bit of a hack because I need to install a specific git hash: the fix is not yet released.

I'm not familiar w/ bioc so I don't know if there's an equivalent command.

Reply:
You can use `BiocManager`. 

It's just `BiocManager::install("rstudio/reticulate", ref="00172079")`

Reply:
Thanks!

I'm not familiar with the implications regarding plain R vs. installing via Bioc.  Should I update this as suggested?

Reply:
I think all the other packages are installed via `BiocManager`. Might as well use the same?

Reply:
ok